compiler/src/dotty/tools/dotc/quoted/QuoteCompiler.scala

```diff
@@ -36,7 +36,7 @@ class QuoteCompiler extends Compiler {
     List(List(new QuotedFrontend))
 
   override protected def picklerPhases: List[List[Phase]] =
-    List(List(new Staging))
+    List(List(new ReifyQuotes))
```

Comment:
Just curious, does it still check phase consistency?

Reply:
This code is there from when we had a single `Staging` phase that did both PCP and reification.
When I split it into `Staging` and `ReifyQuotes` I should have changed it. Unfortunately we did not have a test case that covered it.